docs/helm/eks-deployment.md

```diff
@@ -54,138 +54,27 @@ Set the cluster name in an environment variable that can be reused later:
 EKS_CLUSTER_NAME=my-alfresco-eks
 ```
 
-Create the cluster using a supported version (we are currently testing against
-1.24). Most common choices for instance types are `m5.xlarge` and `t3.xlarge`:
+Create the cluster using the latest supported version, check the main [README](../../README.md).
+Most common choices for instance types are `m5.xlarge` and `t3.xlarge`:
 
 ```sh
-eksctl create cluster --name $EKS_CLUSTER_NAME --version 1.24 --instance-types t3.xlarge
+eksctl create cluster --name $EKS_CLUSTER_NAME --version 1.29 --instance-types t3.xlarge --nodes 3
```

Comment:
Shouldn't it be 1.28 as we state we test only with EKS 1.28?

Reply:
strictly speaking yes, but given that I tested it manually during this PR and we are testing 1.29 on KinD... we could stay with that